src/registrar/assets/js/get-gov-admin.js

```diff
@@ -172,40 +172,39 @@ function addOrRemoveSessionBoolean(name, add){
 ** To perform data operations on this - we need to use jQuery rather than vanilla js. 
 */
 (function (){
-    let selector = django.jQuery("#id_investigator")
-    let assignSelfButton = document.querySelector("#investigator__assign_self");
-    if (!selector || !assignSelfButton) {
-        return;
-    }
-
-    let currentUserId = assignSelfButton.getAttribute("data-user-id");
-    let currentUserName = assignSelfButton.getAttribute("data-user-name");
-    if (!currentUserId || !currentUserName){
-        console.error("Could not assign current user: no values found.")
-        return;
-    }
+    if (document.getElementById("id_investigator") && django && django.jQuery) {
```

Comment:
Relax, this is just an additional check for document.getElementById("id_investigator") && django && django.jQuery) to get rid of a console error. Nothing else in this file.